File: PYTHON/film.py
class Film:
    #konstruktor
    def __init__(self, id:str, judul:str, genre:str, durasi:int, path_gambar:str):
        self.__id = str(id)
        self.__judul = str(judul)
        self.__genre = str(genre)
        self.__durasi = int(durasi)
        self.__path_gambar = str(path_gambar)

    #getter
    def getId(self) -> str:
        return self.__id
    
    def getJudul(self) -> str:
        return self.__judul
    
    def getGenre(self) -> str:
        return self.__genre
    
    def getDurasi(self) -> int:
        return self.__durasi
    
    def getPathGambar(self) -> str:
        return self.__path_gambar

    #setter
    def setId(self, id:str) -> None:
        self.__id = str(id)
        
    def setJudul(self, judul:str) -> None:
        self.__judul = str(judul)
        
    def setGenre(self, genre:str) -> None:
        self.__genre = str(genre)
        
    def setDurasi(self, durasi:int) -> None:
        self.__durasi = int(durasi)
        
    def setPathGambar(self, path_gambar:str) -> None:
        self.__path_gambar = str(path_gambar)

File: PHP/film.php
<?php

class Film {
    //atribut
    private string $id;
    private string $judul;
    private string $genre;
    private int $durasi;
    private string $pathGambar;

    //constructor
    public function __construct(string $id, string $judul, string $genre, int $durasi, string $pathGambar) {
        $this->id = $id;
        $this->judul = $judul;
        $this->genre = $genre;
        $this->durasi = $durasi;
        $this->pathGambar = $pathGambar;
    }

    //getter
    public function getId(): string {
        return $this->id;
    }
    public function getJudul(): string {
        return $this->judul;
    }
    public function getGenre(): string {
        return $this->genre;
    }
    public function getDurasi(): int {
        return $this->durasi;
    }
    public function getPathGambar(): string {
        return $this->pathGambar;
    }

    //setter
    public function setId(string $id): void {
        $this->id = $id;
    }
    public function setJudul(string $judul): void {
        $this->judul = $judul;
    }
    public function setGenre(string $genre): void {
        $this->genre = $genre;
    }
    public function setDurasi(int $durasi): void {
        $this->durasi = $durasi;
    }
    public function setPathGambar(string $pathGambar): void {
        $this->pathGambar = $pathGambar;
    }
}
?>